### DISCUSSION TITLE
issue: Causing validation of input using `trigger` ignores later changes of input

### DISCUSSION THREAD

**Discussion by vladimir-djokic** [react-hook-form/react-hook-form#9276] [Q&A]
### Version Number

7.38.0

### Codesandbox/Expo snack

https://codesandbox.io/s/rhk-trigger-issue-8w2vxh

### Steps to reproduce

1. Go to https://codesandbox.io/s/rhk-trigger-issue-8w2vxh
2. Click on `Save` button, message `REQUIRED` is shown
3. Type anything in the input, input is not being re-validated

![RHK_trigger_issue](https://user-images.githubusercontent.com/16443718/198242972-a06e7fb3-ff7d-490c-850f-c50c13249c0d.gif)

### Expected behavior

Input to be re-validated and error for that input "cleared".

### What browsers are you seeing the problem on?

Firefox, Chrome, Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
you should switch your validation mode to 'onChange', trigger **only** trigger validation once has no other side effect on the input validation itself.

**Reply by vladimir-djokic** [reply]
Is there a way to have validation triggered only for specific list of items and then have re-validation work on `onChange` as with submit?
